---
string label
float32 xDistance
float32 yDistance